docs/guide/build-and-run.md

```diff
@@ -44,6 +44,8 @@ There are some options to `cmake`:
 
 - `-DFT_WITH_CUDA=ON/OFF`: build with/without CUDA (defaults to `ON`).
 - `-DFT_WITH_MKL=<path/to/mkl/root>`: build with MKL (path to MKL is required, defaults to building without it).
+
+  The path accepts by CMake should be a raw unescaped path; i.e. `-DFT_WITH_MKL="/some path"` is good since the quotes are resolved by the shell but `-DFT_WITH_MKL=\"/some\ path\"` is not.
```

Comment:
This sentence seems to be inside a Markdown list. Is it rendered correctly?

Reply:
It is. See [https://github.com/roastduck/FreeTensor/blob/67b2e356c7542580c4896c6b6591999b52ecda06/docs/guide/build-and-run.md](here).

Reply:
![image](https://user-images.githubusercontent.com/6522983/181700262-f245ebb3-1f3c-4349-acb1-2160aa871232.png)

No it's not. @Blealtan